docs/Users_Guide/release-notes.rst

```diff
@@ -9,226 +9,428 @@ When applicable, release notes are followed by the GitHub issue number which des
+       (`#2815 <https://github.com/dtcenter/MET/issues/2815>`_).
+     * SonarQube: Replace "enum" to "enum class"
+       (`#2830 <https://github.com/dtcenter/MET/issues/2830>`_).
+     * Update tag used for the release checksum action
+       (`#2929 <https://github.com/dtcenter/MET/issues/2929>`_).
+     * Enhance the ``unit.py`` MET testing script to allow for expected failures
+       (`#2937 <https://github.com/dtcenter/MET/issues/2937>`_).
+     * Modify configure.ac to define C++17 as the default compilation standard
+       (`#2948 <https://github.com/dtcenter/MET/issues/2948>`_).
+
+  .. dropdown:: METplus-Internal issues
+
+     METplus-Internal is a private repository where internal issues related to
+     security are tracked.
+
+     * MET: Enhance the MET testing framework to provide a mechanism for expected failure
+       (`METplus-Internal#23 <https://github.com/dtcenter/METplus-Internal/issues/23>`_).
+
+  .. dropdown:: METbaseimage testing environment
+
+     METbaseimage is a public repository that provides the Docker environment
+     in which automated testing of MET is performed.
+
+     * Refine METbaseimage to compile dependent libraries from a single tar file
+       (`METbaseimage#9 <https://github.com/dtcenter/METbaseimage/issues/9>`_).
+     * Update METbaseimage to complete the transition to the Debian 12 (bookworm) base image
+       (`METbaseimage#12 <https://github.com/dtcenter/METbaseimage/issues/12>`_).
+     * Enhance METbaseimage to compile the ecKit and Atlas libraries
+       (`METbaseimage#13 <https://github.com/dtcenter/METbaseimage/issues/13>`_).
+     * Enhance METbaseimage to install the YAML Python package
+       (`METbaseimage#15 <https://github.com/dtcenter/METbaseimage/issues/15>`_).
+     * Enhance METbaseimage to install SciPy Python package needed by the MET TC-Diag tool
+       (`METbaseimage#20 <https://github.com/dtcenter/METbaseimage/issues/20>`_).
+     * Fix METbaseimage environment to correct the ncdump runtime linker error
+       (`METbaseimage#24 <https://github.com/dtcenter/METbaseimage/issues/24>`_).
+     * Update METbaseimage to use newer versions of Atlas and ecKit
+       (`METbaseimage#27 <https://github.com/dtcenter/METbaseimage/issues/27>`_).
 
-  .. dropdown:: Bugfixes
+MET Upgrade Instructions
+========================
 
-     * Bugfix: Fix Point2Grid processing of GFS Ocean data input (`#2936 <https://github.com/dtcenter/MET/issues/2936>`_).
-     * **Bugfix: Fix contingency table statistic bugs in the CTS and NBRCTS line types for BAGSS, SEDI CI's, ORSS, and ORSS CI's** (`#2958 <https://github.com/dtcenter/MET/issues/2958>`_).
-     * Bugfix: Fix the grid dimensions used for `point2grid_cice_set_attr_grid` unit test (`#2968 <https://github.com/dtcenter/MET/issues/2968>`_).
-     * Bugfix: Fix MTD to run on any MET-supported grid projection (`#2979 <https://github.com/dtcenter/MET/issues/2979>`_).
+MET Version 12.0.0 Upgrade Instructions
```

Comment:
I suggest moving the "MET Version 12.0.0 Upgrade Instructions" to be in a dropdown menu similar to what is below for "Configuration file changes since version 11.1.0" and "Output line type changes since version 11.1.0".